mandatory/2-function-creation.js

```diff
@@ -4,7 +4,13 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  standardAge = 18;
+  loggedIn = true;
```

Comment:
Is this the correct way to define your variables?

Reply:
Sorry should have left you more detailed comment.

Reply:
Done!